Docs 7695 pm 2.3 troubleshooting enhancements mc (#82)
mulesoft/docs-partner-manager: modules/ROOT/pages/ts-connectivity-system.adoc

> = Troubleshooting Connectivity or System Related Errors

Reviewer: ```suggestion
= Troubleshooting Connectivity or System-Related Errors
```

Author: Fixed.

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> If you encounter problems receiving or sending data, the messages and solutions in this section can help you resolve your issue.

Reviewer: ```suggestion
If you encounter problems receiving or sending data, the messages and solutions described here can help you resolve your issue.
```
